services/buildbot/master/master.cfg

```diff
@@ -6,17 +6,6 @@
 # <http://buildbot.twistedmatrix.com/>. Passwords and other secret
 # information are loaded from a neighboring file called 'private.py'.
 
-# Enable TCP keepalives.  The build master is terrible at noticing
```

Comment:
True. Thanks.

I have updated it and made a ticket in Twisted as this smells bad to me... we are monkey patching Twisted :)